cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -701,6 +701,25 @@ def _validate(self, compute_subnet_ids, new_storage_count):
             )
 
 
+class UnmanagedFsxMultiAzValidator(Validator):
+    """
+    Unmanaged FSx Storage Vs Multiple Subnets validator.
+
+    Unmanaged FSx volumes can exist in AZ that are different from the ones defined in queues configuration.
+    In these cases we notify customers that they may incur in increased latency and costs.
+    """
+
+    def _validate(self, queues, fsx_az_list):
+        for queue in queues:
+            if not set(queue.networking.az_list) == set(fsx_az_list):
+                self._add_failure(
+                    "Your configuration for Queue {0} includes multiple subnets and external shared storage "
```

Comment:
quote the values, i.e. `'{0}'`

Reply:
I had quotes. I removed them since in the messages reported in the doc the values were unquoted.

Anyway I'll bring back the quotes.